### ISSUE TITLE
Could diploS/HIC detect background selection?

### ISSUE THREAD

**Issue by jiangzy26** [kr-colab/diploSHIC#57]
Hi, I am looking for a method to quantify the effect of background selection along the genome in 20 diploid samples, I am wondering could diplos/HIC do this analysis. Could you share your help? Thank you very much.

**Comment by andrewkern** [comment]
no, diploshic isn't appropriate for detecting background selection. it is for detecting selective sweeps only.
